nuxt.config.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
 			process.env.JWT_SHOW_TIMEOUT_WARNING_SECONDS,
 		JWT_TIMEOUT_SECONDS: process.env.JWT_TIMEOUT_SECONDS,
 		SC_THEME: process.env.SC_THEME,
+		SENTRY_SAMPLE_RATE: parseFloat(process.env.SENTRY_SAMPLE_RATE) || 1.0,
```

Comment:
parsefloat does not work for undefined (if the value is not defined). 
```suggestion
		SENTRY_SAMPLE_RATE: process.env.SENTRY_SAMPLE_RATE ?parseFloat(process.env.SENTRY_SAMPLE_RATE) : 1.0,
```

Reply:
nice ;) done